src/components/Api.js

```diff
@@ -0,0 +1,90 @@
+export default class Api {
+  constructor(options) {
+    this._url = options.url;
+    this._headers = options.headers;
+  }
+  _getResponse(res) {
+    return res.ok ? res.json() : Promise.reject(`Error: ${res.status}`);
+  }
+
+  getUserInfo() {
+    return fetch(this._url + "users/me", {
+      headers: this._headers,
+    }).then(this._getResponse);
+  }
+
+  getInitialCards() {
+    return fetch(this._url + "cards", {
+      headers: {
+        authorization: "0d8f734d-caf1-45e3-b9d3-764b4099955a"
```

Comment:
According to the task you need to pass full `headers` into the constructor of `Api`. See here https://disk.yandex.ru/i/J93dbJp7eypXKQ . And in the code you need to use only: `headers: this._headers` (not to duplicate the code)